src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-tabs/smb-tabs.component.ts

```diff
@@ -1,8 +1,10 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
 
+const SMB_PATH = '/cephfs/smb/';
```

Comment:
```suggestion
const SMB_PATH = '/cephfs/smb';
```
See another following comment

Reply:
Changed it to `cephfs/smb` following other patterns as well